src/metabase/pulse/render/js_svg.clj

```diff
@@ -115,7 +116,8 @@
   [rows labels settings]
   (let [svg-string (.asString (js/execute-fn-name @context "timeseries_waterfall" rows
                                                   (map (fn [[k v]] [(name k) v]) labels)
-                                                  (json/generate-string settings)))]
+                                                  (json/generate-string settings)
+                                                  (json/generate-string (public-settings/application-colors))))]
```

Comment:
Looks good. Keep in mind that `public-settings/application-colors` may be an empty map, and if it has values, it _only_ has values which are different than defaults. For instance, if only the 'brand' color was changed to `#00FF00` in the admin panel, then this map will look like: `{:brand "#00FF00"}`, with no other colors set.

I think the changes you've made in this PR will properly get colors anyway, just wanted to make that clear about the setting coming from the backend.